### PR TITLE
Add blog about using Grafana creating dashboards based on DWH

### PR DESCRIPTION
**Pull request by sradco**
I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sradco

This pull request needs review by: @mykaul @yanivlavi

### CODE REVIEW

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+---
+title: Build oVirt Report Using Grafana
```
Comment (mykaul) [inline source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: Report -> Reports

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+author: sradco,
+tags: oVirt, oVirt 4.2, open source virtualization, report, dwh
+date: 2018-01-16 09:00:00 CET
+---
+
+[Grafana](https://grafana.com/), The open platform for beautiful analytics and monitoring,
+adds support for [PostgreSQL](http://docs.grafana.org/features/datasources/postgres/).
```
Comment (mykaul) [inline source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: adds -> recently added

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+It in now possible to connect Grafana to [oVirt DWH](https://www.ovirt.org/documentation/how-to/reports/dwh/),
+in order to visualize and monitor the oVirt environment.
+
+**Grafana dashboard example**
+![](/images/grafana_dashboard_example.png)
+
+READMORE
+
+If you wish to create dashboards to monitor Ovirt environment, you will need to [install Grafana](http://docs.grafana.org/installation/rpm/).
```
Comment (mykaul) [inline source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: Ovirt -> oVirt

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+Grafana automatically create and admin [user](http://docs.grafana.org/installation/configuration/#admin-user) and [password](http://docs.grafana.org/installation/configuration/#admin-password).
```
Comment (mykaul) [inline source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: create -> creates
and admin -> an admin

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+
+
+**Graph panel example:**
+
+To add a `Graph` type panel, on the left side you have the [Row controls menu](http://docs.grafana.org/guides/getting_started/#dashboards-panels-rows-the-building-blocks-of-grafana).
+Go to the `+ Add Panel`, and pick `Graph`.
+
+Query example for the - Five Most Utilized Hosts by Memory / CPU:
+
```
Comment (mykaul) [inline source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: There's a markdown for raw text?

Comment (sradco) [reply source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: What do you mean?

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md (hunk 4 above)
Comment (sradco) [reply source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: Fixed

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md (hunk 3 above)
Comment (sradco) [reply source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: Fixed

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md (hunk 2 above)
Comment (sradco) [reply source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: Fixed

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md (hunk 1 above)
Comment (sradco) [reply source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: Fixed

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md (hunk 5 above)
Comment (mykaul) [reply source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code or https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#blockquotes

File: source/blog/2018-01-16-ovirt-report-using-grafana.html.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+    stats_hosts.host_id,
+    host_name,
+    delete_date,
+    history_datetime
+) AS a
+GROUP BY a.host_name, a.mem_usage
+ORDER BY time
+```
+
+The query uses the [Templateing](http://docs.grafana.org/reference/templating/) feature, to enable input controls.
```
Comment (mykaul) [inline source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: Templateing -> Templating

Comment (sradco) [reply source/blog/2018-01-16-ovirt-report-using-grafana.html.md]: Done. Thanks.

**Pull request merged**